EXAMPLE.md

```diff
@@ -19,83 +21,99 @@ sed -i -e 's/use_avahi    = false/use_avahi    = true/' examples/main.tf-jenkins
 ```
 
 Also adjust the email to receive the reports. You should use an address local to your machine (`yourusername@localhost`)
+
 ```bash
 sed -i -e 's/default = "juliogonzalez@localhost"/default = "YOUREMAIL"/' examples/main.tf-jenkins.mycopy
 ```
 
 Export SCC credentials:
+
 ```bash
 export TF_VAR_SCC_USER=<ID>
 export TF_VAR_SCC_PASSWORD=<PASSWORD>
 ```
 
 Export a variable `BUILD_TIMESTAMP` to be used by all the steps (when running on Jenkins this is not needed and `BUILD_NUMBER` provided by Jenkins can be used automatically):
+
 ```bash
 export BUILD_TIMESTAMP=$(date '+%Y-%m-%d-%H-%M-%S')
 ```
 
 Create a folder for the outputs (not required if ìt already exists):
+
 ```bash
 mkdir /tmp/sumaform_outputs
 ```
 
 Clone or update the repository with the terraform code (sumaform in our case):
-```
+
+```bash
 ./terracumber-cli --tf examples/main.tf-jenkins.mycopy --outputdir /tmp/sumaform_outputs --gitrepo https://github.com/uyuni-project/sumaform.git --gitfolder /tmp/sumaform --gitref master --runstep gitsync
 ```
 
 Create the terraform environment:
-```
+
+```bash
 ./terracumber-cli --tf examples/main.tf-jenkins.mycopy --outputdir /tmp/sumaform_outputs --logfile /tmp/sumaform/sumaform.log  --gitfolder /tmp/sumaform --init --runstep provision
 ```
 
-Run an Uyuni testsuite step (iterate for more steps if needed):
-```
+Run an Uyuni test suite step (iterate for more steps if needed):
```

Comment:
In which cases do I need to iterate for more steps? I know what you mean, but I would like at least to get some redirection here to the specific location in the repository.